test/cowboy_swagger_test_utils.erl

```diff
@@ -8,7 +8,9 @@
       headers => [tuple()],
       body => binary()}.
 -type config() :: proplists:proplist().
+-type shotgun_http_verb() :: delete | get | head | options | patch | post | put.
```

Comment:
Maybe `shotgun` should also expose these?

Reply:
It would be a great idea, yes.